portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -0,0 +1,42 @@
+const getComments = async (limit = 5) => {
```

Comment:
[nit] It's good practice to make all hard-coded values into variables like `const commentLimit = 5`. 

It's best to always have it available as a variable to avoid another piece of code also hard-coding "5" to match, and then one day someone updates it here, but not in the other piece of code.